content/docs/concepts/traffic-management/network/index.md

```diff
@@ -0,0 +1,325 @@
+metadata:
+  name: ratings
+spec:
+  hosts:
+    - ratings
+  http:
+  - route:
+    - destination:
+        host: ratings
+        subset: v1
+    retries:
+      attempts: 3
+      perTryTimeout: 2s
+{{< /text >}}
+
+Consumers of a service can also override timeout and retry defaults with
+request-level overrides through special HTTP headers. The Envoy proxy
+implementation makes the following headers available:
+
+-  Timeouts: `x-envoy-upstream-rq-timeout-ms`
+
+-  Retries: `X-envoy-max-retries`
+
+## Circuit breakers
+
+As with timeouts and retries, you can configure a circuit breaker pattern
+without changing your services. While retries let your application recover from
+transient errors, a circuit breaker pattern prevents your application from
+stalling as it waits for an upstream service to respond. By configuring a
+circuit breaker pattern, you allow your application to fail fast and handle the
+error appropriately, for example, by triggering an alert. You can configure a
+simple circuit breaker pattern based on a number of conditions such as
+connection and request limits.
+
+### Limit connections to 100
+
+The following destination rule sets a limit of 100 connections for the
+`reviews` service workloads of the v1 subset:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews
+spec:
+  host: reviews
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+    trafficPolicy:
+      connectionPool:
+        tcp:
+          maxConnections: 100
+{{< /text >}}
+
+See the [circuit-breaking task](/docs/tasks/traffic-management/circuit-breaking/)
+for detailed instructions on how to configure a circuit breaker pattern.
+
+## Health checking
+
+Istio currently supports passive health checking to minimize the chances of
+accessing an unhealthy instance in the load balancing pool. The Envoy proxies
+support active health checking but Istio doesn’t enable it. When you combine
+Istio health checks with platform-level health checks, such as those supported
+by Kubernetes, your applications can quickly eject unhealthy pods, containers,
+or VMs from the service mesh, minimizing request failures and impact on
+latency.
+
+The Envoy proxies follow a circuit breaker pattern to classify instances as
+unhealthy or healthy based on their failure rates for the health check API
+call. The process to remove unhealthy instances from the load balancing pool is
+independent of platform:
+
+1. Envoy periodically checks the health of each instance in the load balancing
+   pool.
+
+1. When the number of failed health checks for a given instance exceeds a
+   specified threshold, the Envoy proxy removes the instance from the service's
+   load balancing pool.
+
+1. When the number of passed health checks exceeds a specified threshold, the
+   Envoy proxy adds the instance back into the service's load balancing pool.
+
+The following diagram shows steps 1-3 in the process of removing an unhealthy
```

Comment:
How so? I don't see anything in the diagram that indicates an instance being checked, removed, or added.

Reply:
The diagram has numbers in gray, they correspond to the numbers in the process.

Reply:
Yes, I see the numbered arrows, but I don't understand how those arrows can be interpreted to be showing check, remove, and add instances to the lb pool of an Envoy.

Reply:
@frankbu What would you want to see? I tried to keep the diagram as simple as possible. We shouldn't add the text to the diagram to ensure the document is accessible to screen readers.

Reply:
Maybe a diagram that shows 3 pictures. 1. envoy pointing to 2 instances of another service with one of the instances highlighted as bad health check, 2. envoy pointing to 1 instance, the other one has the arrow removed. 3. envoy pointing to the 2 instances again, this time the bad instance is no longer highlighted as bad.

Something like that, anyway. That said, I don't think a diagram is really needed here, and especially one that doesn't really show the steps like it says it does, so I would just remove the diagram.